Packs/ServiceNow/ReleaseNotes/2_7_6.md

```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+
+##### New: ServiceNowTroubleshoot
+
+- Troubleshot ServiceNow v2 integration by displaying health information of enabled instances and old and active incidents of disabled instances.
```

Comment:
Troubleshoot the ServiceNow v2 integration by displaying the health information of enabled instances along with old and active incidents of disabled instances.